File: app/src/main/java/io/temco/guhada/data/model/NaverResponse.java
package io.temco.guhada.data.model;

import com.google.gson.annotations.SerializedName;

public class NaverResponse {
    private String message;

    @SerializedName("resultcode")
    private String resultCode;

    @SerializedName("response")
    private NaverUser user;

    public String getResultCode() {
        return resultCode;
    }

    public void setResultCode(String resultCode) {
        this.resultCode = resultCode;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public NaverUser getUser() {
        return user;
    }

    public void setUser(NaverUser user) {
        this.user = user;
    }
}

File: app/src/main/java/io/temco/guhada/data/model/naver/NaverResponse.kt
package io.temco.guhada.data.model.naver

import com.google.gson.annotations.SerializedName

/**
 * 네아로 Response 클래스
 * @see NaverUser
 * @author Hyeyeon Park
 */
class NaverResponse {
    var message: String = ""

    @SerializedName("resultcode")
    var resultCode: String = ""

    @SerializedName("response")
    var user: NaverUser = NaverUser()
}